graderx/graders/manager.py

```diff
@@ -131,80 +156,97 @@ def compressed_results(course_name, lab):
     zip_file_path = helpers.create_zip_file(results_files)
     return zip_file_path
 
+
 def get_diff_results_file(course_name, lab):
-    path = Path("graderx").joinpath("graders").joinpath("courses").joinpath(course_name).joinpath(lab)
+    path = Path("graderx").joinpath("graders").joinpath(
```

Comment:
I guess path supports the `/` overloaded operator so I *guess* this can be `Path("graderx") / "graders" / "courses"`

but that's for later, it's not a problem